ui/app/AppLayouts/Wallet/views/RightTabView.qml

```diff
@@ -148,7 +148,7 @@ RightTabBaseView {
                             id: assetsViewAdaptor
 
                             accounts: RootStore.addressFilters
-                            chains: RootStore.networkFilters
+                            chains: RootStore.networkFilters.split(":")
```

Comment:
it could be split already in `RootStore` because wherever it's used, it's always split first.